### DISCUSSION TITLE
Why New-ServiceNowChangeTask does not link the new task to the main change ticket?

### DISCUSSION THREAD

**Discussion by pheresparza** [Snow-Shell/servicenow-powershell#254] [General]
Hello folks, 
Before anything would like to thanks all the support you guys provide... by other hand, I have been trying to accomplish the change ticket creation using this amazing PowerShell module however,  when playing with New-ServiceNowChangeTask using the parameter -ChangeRequest CHGXXXXXX I get the ctask created but it is not linked to the CHGXXXXXX ticket, I see it as independent ctask. Wonder if I am doing anything wrong? thoughts

**Comment by gdbarron** [comment]
Hi @pheresparza.  Thanks for bringing this up and you weren't doing anything wrong...it was a bug.  This has been fixed and v4.0.3 has been released.

**Reply by pheresparza** [reply]
Thanks for the fix... I just found a work around to this situation but event better this has now a normal processing
